resources/lang/en/forum.php

```diff
@@ -127,6 +127,49 @@
             'previous' => 'go back 10 posts',
         ],
 
+        'logs' => [
+            '_' => 'Topic logs',
+            'title' => 'topic logs',
+
+            'columns' => [
+                'action' => 'Action',
+                'date' => 'Date',
+                'ip' => 'IP Address',
+                'user' => 'User',
+            ],
+
+            'data' => [
+                'add_tag' => 'added ":data" tag',
+                'announcement' => 'pinned topic and marked as announcement',
+                'delete_post' => 'from topic :data',
```

Comment:
an actual post id (and link to it) would be more useful. Same with `post_edited` and `restore_post`.

Reply:
post id isn't stored though :thinking:

should i make it a thing?

Reply:
wtf the log function in the controller is a lie :raised_eyebrow: 

see if it's easy to add

Reply:
yeah was confused about that as well 😔, probably an easy change though